src/Magnum/Audio/Test/RendererTest.cpp

```diff
@@ -46,6 +66,63 @@ void RendererTest::debugError() {
     CORRADE_COMPARE(out.str(), "Audio::Renderer::Error::InvalidOperation\n");
 }
 
+void RendererTest::debugDistanceModel() {
+    std::ostringstream out;
+    Debug(&out) << Renderer::DistanceModel::Inverse;
+    CORRADE_COMPARE(out.str(), "Audio::Renderer::DistanceModel::Inverse\n");
+}
+
+void RendererTest::listenerOrientation() {
+    constexpr Vector3 up{1, 2, 3}, fwd{3, 2, 1};
```

Comment:
Please use `{1.0f, 2.0f, 3.0f}` instead of integers. In combination with the {}-style initialization it causes implicit conversion warnings/errors on some compilers.